### DISCUSSION TITLE
Purchase different tickets at the same time.

### DISCUSSION THREAD

**Discussion by rickcj** [pretix/pretix#4039] [Q&A]
Is it possible to add tickets from different events to the cart and then checkout in one go?

We are selling tickets to entertainment for our community centre and its a pain for people to buy tickets to different events as they have to repeatedly go through the checkout for each event individually.

**Comment by pc-coholic** [comment]
Carts cannot be mixed across events.

But - depending on your setup, circumstances and requirements - you can mix products from multiple dates from within a single event-series.

**Reply by rickcj** [reply]
Shame as it restricts and complicates it.

**Reply by pc-coholic** [reply]
It would be nice, for sure... But I don't want to give you any false hope: it won't happen...

From a technical point of view, every event (or event-series) is scoped/a closed system. And since payment providers are not configured on the organizer level but rather the event level, joining multiple events into one cart is pretty much impossible. Same goes for event settings, payment deadlines, tax rules, etc.

Just as an example: if you have an event that offers only credit card payments with a 2 week deadline and another event that only offers on site payment with a 2 day deadline: how can we reconcile this? It probably has to be some kind of "least restrictive" or "most restrictive". This might work for numeric settings like a payment deadline. But pretix has no concept of what is more or less restrictive, when it comes to payment methods/payment types.

And if a event organizer has configure different merchant for a payment provider... Which account gets the payment?

This is a limitation of the way pretix is designed - but unfortunately not one, that can be worked around easily...
